examples/shell/nrfconnect/boards/nrf52840dk_nrf52840.overlay

```diff
@@ -0,0 +1,27 @@
+/*
+ *    Copyright (c) 2020 Project CHIP Authors
```

Comment:
2020 -> 2021

Reply:
That's stupid.. I hope soon the dates will be removed from the copyright header :)